### ISSUE TITLE
Supervisor has dropped the use asynchat as it's been removed on python 3.12

### ISSUE THREAD

**Issue by EraYaN** [alexsilva/supervisor#53]
Please see if you can pull the changes as well, so this will run on 3.12 and later.

**Comment by alexsilva** [comment]
I have been regularly updating the project. I will make new updates soon, thank you.

**Comment by alexsilva** [comment]
I couldn't see any major changes other than declaring compatibility with Python 3.11 and 12.

https://github.com/alexsilva/supervisor/compare/windows...Supervisor%3Asupervisor%3Amain

**Comment by EraYaN** [comment]
It happened in 4.2.5 as listed in the changelog. They seem to have vendored some version of asynchat.

https://github.com/Supervisor/supervisor/commit/63729959eab7f02c888e9b76a9ab0c2e1b668ed1 seems to be this commit. (PR: https://github.com/Supervisor/supervisor/issues/1520)

Which seems to already be in this repo, which makes it weird that I got an import error. I'll have to test again tomorrow on a fresh 3.12 venv. Seems to currently not do anything when I try it now and just hang up starting, but this a different machine.

EDIT: I have tried with a fresh 3.12 conda environment and it all seems to work just fine in with the basic settings and I have no idea how I even got to this issue, so for now I'll close it.
